<!DOCTYPE html>
<html lang="en-US">
  <head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Hufeng Zhou Official Personal Website</title>
    <link rel="stylesheet" href="css/components.css">
    <link rel="stylesheet" href="css/icons.css">
    <link rel="stylesheet" href="css/responsee.css">
    <link rel="stylesheet" href="owl-carousel/owl.carousel.css">
    <link rel="stylesheet" href="owl-carousel/owl.theme.css">     
    <link rel="stylesheet" href="css/template-style.css">
    <link href='https://fonts.googleapis.com/css?family=Playfair+Display&subset=latin,latin-ext' rel='stylesheet' type='text/css'>
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,700,800&subset=latin,latin-ext' rel='stylesheet' type='text/css'>
    <script type="text/javascript" src="js/jquery-1.8.6.min.js"></script>
    <script type="text/javascript" src="js/jquery-ui.min.js"></script>      
  </head>
  
  <body class="size-1140">
    <!-- HEADER -->
    <header role="banner" class="position-absolute">    
      <!-- Top Navigation -->
      <nav class="background-transparent background-transparent-hightlight full-width sticky">
        <div class="s-12 l-2">
          <a href="index.html" class="logo">
            <!-- Logo White Version -->
            <img class="logo-white" src="img/logo-dark.png" alt="">
            <!-- Logo Dark Version -->
            <img class="logo-dark" src="img/logo-dark.png" alt="">
          </a>
        </div>
        <div class="top-nav s-12 l-10">
          <p class="nav-text"></p>
          <ul class="right chevron">
            <li><a href="index.html">Home</a></li>
	    <li><a href="index.html">About</a></li>
            <li><a href="publications.html">Publications</a></li>
<li>
  <a>Projects</a>
  <ul>
    <li>
      <a href="Epigenomics.html">Epigenomics</a>
      <ul>
        <li>
          <a href="EBV_Oncoproteins.html">EBV oncoproteins</a>
        </li>
        <li>
          <a href="EBV_Super_Enhancers.html">Super Enhancers</a>
        </li>
        <li>
          <a href="EBV_Regulome.html">EBV Regulome</a>
        </li>
      </ul>
    </li>
	  
    <li>
      <a href="Population_Genetics.html">Population Genetics</a>
      <ul>
        <li>
          <a href="FAVOR.html">Functional Annotation Database</a>
        </li>
        <li>
          <a href="STAAR.html">STAAR</a>
        </li>
      </ul>
    </li>

    <li>
      <a href="PPIs.html">Protein-Protein Interactions</a>
      <ul>
        <li>
          <a href="IntraPPIs.html">Intra-Species PPIs</a>
        </li>
        <li>
          <a href="Host-Pathogen-PPIs.html">Host-Pathogen PPIs</a>
        </li>
        <li>
          <a href="Protein-functions.html">Protein Functions</a>
        </li>	      
      </ul>
    </li>
	  
  </ul>
</li>
		
		
            
            <li><a href="index.html">Software</a></li>
            <li><a href="index.html">Contact</a></li>
          </ul>
        </div>
      </nav>
    </header>
    
    <!-- MAIN -->
    <main role="main">    
      <!-- Main Header -->
      <header>
        <div class="carousel-default owl-carousel carousel-main carousel-nav-white background-dark text-center">
          <div class="item">
            <div class="s-12">
              <img src="img/header.jpg" alt="">
              <div class="carousel-content">
                <div class="content-center-vertical line">
                  <div class="margin-top-bottom-80">
                    <!-- Title -->
                    <h1 class="text-white margin-bottom-30 text-size-60 text-m-size-30 text-line-height-1">Hufeng Zhou<br> Official Website</h1>
                    <div class="s-12 m-10 l-8 center"><p class="text-white text-size-14 margin-bottom-40">This is the official website introduce the research and work.</p></div>
                    <div class="line">
                      <div class="s-12 m-12 l-3 center">
                       <!--  <a class="button button-white-stroke s-12" href="/">Get Started Now</a>-->
                      </div>       
                    </div>  
                  </div>
                </div>
              </div>
            </div>
          </div>              
        </div>               
      </header>
      
      <!-- Section 1 -->
      <section class="section-small-padding background-white text-center"> 
        <div class="line">
          <div class="margin">
            <div class="s-12 m-12 l-4 margin-m-bottom">
              <div class="padding-2x block-bordered">
                <i class="icon-sli-shield icon3x text-dark center margin-bottom-30"></i>
                <h2 class="text-thin">Publications</h2>
                <p class="margin-bottom-30">Peer reviewed publications of my work on world's leading journals.</p>
                <a class="button button-dark-stroke text-size-12" href="/">GET MORE INFO</a>
              </div>
            </div>
            <div class="s-12 m-12 l-4 margin-m-bottom">
              <div class="padding-2x block-bordered">
                <i class="icon-sli-umbrella icon3x text-dark center margin-bottom-30"></i>
                <h2 class="text-thin">Eductional and Training</h2>
                <p class="margin-bottom-30">More than a decade of experience and training in computational biology.</p>
                <a class="button button-dark-stroke text-size-12" href="/">GET MORE INFO</a>
              </div>
            </div>
            <div class="s-12 m-12 l-4 margin-m-bottom">
              <div class="padding-2x block-bordered">
                <i class="icon-sli-home icon3x text-dark center margin-bottom-30"></i>
                <h2 class="text-thin">Projects Experience</h2>
                <p class="margin-bottom-30">Detail explanation of my project experience.</p>
                <a class="button button-dark-stroke text-size-12" href="/">GET MORE INFO</a>
              </div>
            </div>
          </div>
        </div>
      </section>
      
      <!-- Section 1 -->
      <section class="section">
        <div class="line">
          <h2 class="text-size-50 text-center">About</h2>
          <hr class="break-small background-primary break-center">
        </div>
        <div class="line">
          <div class="margin">  
            <div class="s-12 m-12 l-2">
              <p class="h1 text-size-30 margin-m-bottom-30 text-primary text-line-height-1 text-right">Dr. Zhou is <br> a scientist and an engineer.</p>
            </div>
            <div class="s-12 m-12 l-5">
              <p class="text-size-14 margin-m-bottom-30 text-dark"> Dr. Hufeng Zhou is an Instructor (Junior Faculty Member) at Harvard Medical School and Research Scientist in Harvard T. H. Chan School of Public Health with more than 15 years of expertise in exploring fundamental epigenetic mechanisms of EBV-associated cancers using Next-Generation Sequencing technologies. He is world-renowned for his expertise in human genetics, particularly in rare variant association studies, functional annotation of human genetics variants, and quality control of super large scale of whole genome/exome sequencing studies.  </p>
            </div>
            <div class="s-12 m-12 l-5">
              <p class="text-size-14 text-dark"> Dr. Zhou's broad research interests also involves investigating the regulatory effects of EBV super-enhancers through non-coding RNA and long-range interactions, using 3D genomics and functional information to identify disease-related variants, and applying machine learning strategies to clinical data. Additionally, he has years of research experience in protein-protein interactions, pathway data integration and analysis, human-microbiome metagenomic data analysis, and software and database development.</p>
            </div>
          </div>
        </div>    
      </section>
      
      <!-- Section 2 --> 

<section class="full-width">
  <div class="s-12 m-12 l-6">  
    <img class="my-image" src="img/me.png" alt="">
  </div>
  <div class="s-12 m-12 l-6">
    <div class="padding-2x">
      <div class="line">
        <div class="float-left">
          <i class="icon-sli-drop icon3x text-dark"></i>
        </div>
        <div class="margin-left-60 margin-bottom-30">
          <h3 class="text-size-25 margin-bottom-0">Mt. Sinai Hospital</h3>
          <!--the following paragraph is not wrapped properly-->

          <p class="text-dark wrap-description">This picture was taken at the most special moment in my life, when I was presenting my work on the FAVOR functional annotation database at a conference at Mt. Sinai Hospital. After my presentation, I took a walk around the Central Park nearby, and that was the start of my whole new life. It was a moment of immense pride and satisfaction, having the opportunity to share my work and ideas with experts in the field. The experience was truly unforgettable and reinforced my passion for pursuing a career in this field.<br> Feb 2020.</p>            
        </div>
      </div>
    </div>
  </div>
</section>

	    
<style>
.my-image {
    width: 46%;
    float: right;
  }
	
.wrap-description {
    max-width: 46%; /* Adjust this to match the image width */
  }
</style>
	    <section class="section">
		    
            <div class="line">
              <div class="float-left">
                <i class="icon-sli-screen-smartphone icon3x text-dark"></i>
              </div>
              <div class="margin-left-60 margin-bottom-30">
                <h3 class="text-size-25 margin-bottom-0">Computational Biologist and Engineer</h3>
                <p class="text-dark"> Dr. Zhou is a highly accomplished Computational Biologist and Engineer, well-versed in conducting cutting-edge scientific research in the field of human genomics. In addition to his research pursuits, Dr. Zhou has made notable contributions to advancing the study of human genomics through his development of various innovative tools and technologies. For instance, he has designed and implemented pathway databases such as IntPath, functional annotation databases like FAVOR, and novel algorithms for predicting host-pathogen interactions. Dr. Zhou has also developed tools to construct novel regulatory networks of transcriptome, further demonstrating his expertise in this field..</p>            
              </div>
            </div>
            
            <div class="line">
              <div class="float-left">
                <i class="icon-sli-heart icon3x text-dark"></i>
              </div>
              <div class="margin-left-60 margin-bottom-30">
                <h3 class="text-size-25 margin-bottom-0">Passionate Trail Biker and Rower</h3>
                <p>Dr. Zhou is an accomplished mountain biker with a particular interest in cross-country trail biking and an avid rower. Despite engaging in intensive programming and research throughout the day, he finds solace in exploring cross-country trails in the nearby woods and reserves, as well as rowing down the Charles River from MIT to Harvard on sunny days. Engaging in these sports promotes both physical and mental well-being and keeps Dr. Zhou enthusiastic, energized, and level-headed in his competitive career.</p>            
              </div>
            </div>
            
            <div class="line">
              <div class="float-left">
                <i class="icon-sli-chart icon3x text-dark"></i>
              </div>
              <div class="margin-left-60 margin-bottom-30">
                <h3 class="text-size-25 margin-bottom-0">Accomplished Scientist</h3>
                <p>Dr. Zhou is a distinguished scientist with over 15 years of dedicated and fruitful experience in the fields of computational biology and biostatistics, with a particular focus on human genetics. He has made significant contributions to the study of human population genetics, functional annotation of human genetic variants, epigenomics and chromatin confirmation, and proteomics of human and microbes. Dr. Zhou's research has been widely cited and has had a profound impact on the fields in which he works.</p>            
              </div>
            </div>
            
            <div class="line">
              <div class="float-left">
                <i class="icon-sli-globe-alt icon3x text-dark"></i>
              </div>
              <div class="margin-left-60">
                <h3 class="text-size-25 margin-bottom-0">Global and Inclusive Mindset</h3>
                <p class="text-dark">Dr. Zhou's mission is to advance humanity through his research, focusing on better understanding human genetic diseases, drug targets for infectious diseases, and developing more sensitive and effective screening technologies. He lives by the motto,  "To ordain conscience for Heaven and Earth. To secure life and fortune for the populace. To continue lost teachings for past sages. To establish peace for all future generations." (为天地立心，为生民立命，为往圣继绝学，为万世开太平。)</p>            
              </div>
            </div> 
          </div>
        </div>
      </section>
      
      <!-- Section 3 -->
      <section class="section background-white" >  
        <div class="line">
          <h2 class="text-size-50 text-center">Stats</h2>
          <hr class="break-small background-primary break-center">
          <div class="margin margin-top-bottom-50">
            <div class="s-12 m-12 l-3">
              <div class="block">
                <div class="count-to">
                  <span class="timer h1 text-size-50">2166</span>
                  <p class="h1 text-size-20 margin-top-10 text-dark text-thin">Citations to my publications</p> 
                </div>
              </div>
            </div>
            <div class="s-12 m-12 l-3">
              <div class="block">
                <div class="count-to">
                  <span class="timer h1 text-size-50">38</span>
                  <p class="h1 text-size-20 margin-top-10 text-dark text-thin">Peer Reviewed Research Articles</p> 
                </div>
              </div>
            </div>
            <div class="s-12 m-12 l-3">
              <div class="block">
                <div class="count-to">
                  <span class="timer h1 text-size-50">26</span>
                  <p class="h1 text-size-20 margin-top-10 text-dark text-thin">h-index</p> 
                </div>
              </div>
            </div>
            <div class="s-12 m-12 l-3">
              <div class="block">
                <div class="count-to">
                  <span class="timer h1 text-size-50">15 years</span>
                  <p class="h1 text-size-20 margin-top-10 text-dark text-thin">Dedicated Experience</p> 
                </div>
              </div>
            </div> 
          </div>
        </div>
      </section>
      
      <hr class="break margin-top-bottom-0">
      
      <!-- Section 4 -->
      
     <!--reduce the size of image-testimonial-small-->


     <section class="section background-white text-center">
      <div class="line">
        <h2 class="text-size-50 text-center">Selected Work</h2>
        <hr class="break-small background-primary break-center">
        <div class="carousel-default owl-carousel carousel-wide-arrows">
          <div class="item">
            <div class="s-12 m-12 l-7 center text-center">
              <img class="image-testimonial-small" src="img/FAVOR.png" alt="">
              <p class="h1 margin-bottom text-size-20">FAVOR integrates variant functional information from multiple sources to describe the functional characteristics of variants and facilitates prioritizing plausible causal variants influencing human phenotypes. Furthermore, we provide a scalable annotation tool, FAVORannotator, to functionally annotate large-scale WGS studies and efficiently store the genotype and their variant functional annotation data in a single file using the annotated Genomic Data Structure (aGDS) format, making downstream analysis more convenient. </p>
              <p class="h1 text-size-16">FAVOR / Nucleic Acids Research / 2023</p>
            </div>
          </div>
          <div class="item"> 
            <div class="s-12 m-12 l-7 center text-center">
              <img class="image-testimonial-small" src="img/STAAR.png" alt="">
              <p class="h1 margin-bottom text-size-20">We propose STAAR (variant-set test for association using annotation information), a scalable and powerful RV association test method that effectively incorporates both variant categories and multiple complementary annotations using a dynamic weighting scheme. For the latter, we introduce ‘annotation principal components’, multidimensional summaries of in silico variant annotations. STAAR accounts for population structure and relatedness and is scalable for analyzing very large cohort and biobank whole-genome sequencing studies of continuous and dichotomous traits. </p>
              <p class="h1 text-size-16">STAAR / Nature Genetics  / 2020</h5>
            </div>
          </div>
          <div class="item">
            <div class="s-12 m-12 l-7 center text-center">
              <img class="image-testimonial-small" src="img/STAARpipe.png" alt="">
              <p class="h1 margin-bottom text-size-20">Detecting rare-variant associations in the noncoding genome is challenging. We present a scalable, flexible and streamlined rare-variant association analysis framework for biobank-scale whole-genome sequencing data, including gene-centric and non-gene-centric analyses by incorporating multiple variant functional annotations using various coding and noncoding units, conditional analysis, result summary and visualization.</p>
              <p class="h1 text-size-16"> STAARpipeline / Nature Methods / 2023</p>
            </div>
          </div>
        </div>
      </div>
    </section>

<style>
.image-testimonial-small {
max-width: 3%;
height: auto;
}
</style> 


     <!-- Section 5 -->        
      <section class="section-top-padding full-width">
        <h2 class="text-size-50 text-center">Research Highlights</h2>
        <hr class="break-small background-primary break-center"> 
        <div class="tabs background-primary-hightlight">
            <div class="tab-item tab-active">

              <a class="tab-label active-btn">Population Genetics</a>

              <div class="tab-content lightbox-gallery">

                <div class="s-12 m-6 l-3">
                  <a class="image-with-hover-overlay image-hover-zoom" href="#">
                    <div class="image-hover-overlay background-primary"> 
                      <div class="image-hover-overlay-content text-center padding-2x">
                        <h2 class="text-thin">STAAR </h2> 
                        <p>We propose STAAR (variant-set test for association using annotation information), a scalable and powerful RV association test method that effectively incorporates both variant categories and multiple complementary annotations using a dynamic weighting scheme. For the latter, we introduce ‘annotation principal components’, multidimensional summaries of in silico variant annotations. STAAR accounts for population structure and relatedness and is scalable for analyzing very large cohort and biobank whole-genome sequencing studies of continuous and dichotomous traits. </p> 
                      </div> 
                    </div> 
                    <img src="img/portfolio/STAAR.png" alt="" title="STAAR" />
                  </a>	
                </div>

                <div class="s-12 m-6 l-3">
                  <a class="image-with-hover-overlay image-hover-zoom" href="#">
                    <div class="image-hover-overlay background-primary"> 
                      <div class="image-hover-overlay-content text-center padding-2x">
                        <h2 class="text-thin">STAARpipeline</h2> 
                        <p>Detecting rare-variant associations in the noncoding genome is challenging. We present a scalable, flexible and streamlined rare-variant association analysis framework for biobank-scale whole-genome sequencing data, including gene-centric and non-gene-centric analyses by incorporating multiple variant functional annotations using various coding and noncoding units, conditional analysis, result summary and visualization.</p> 
                      </div> 
                    </div> 
                    <img src="img/portfolio/STAARpipeline.png" alt="" title="STAARpipeline" />
                  </a>	
                </div>

                <div class="s-12 m-6 l-3">
                  <a class="image-with-hover-overlay image-hover-zoom" href="#">
                    <div class="image-hover-overlay background-primary"> 
                      <div class="image-hover-overlay-content text-center padding-2x">
                        <h2 class="text-thin">FAVORannotator</h2> 
                        <p>FAVORannotator is an R program for performing functional annotation of any genetic study (e.g. Whole-Genome/Whole-Exome Sequencing/Genome-Wide Association Studies) using the FAVOR backend database to create an annotated Genomic Data Structure (aGDS) file by storing the genotype data (in VCF or GDS format) and their functional annotation data in an all-in-one file.</p> 
                      </div> 
                    </div> 
                    <img src="img/portfolio/FAVORannotator.png" alt="" title="FAVORannotator" />
                  </a>	
                </div>
                
                <div class="s-12 m-6 l-3">
                  <a class="image-with-hover-overlay image-hover-zoom" href="#">
                    <div class="image-hover-overlay background-primary"> 
                      <div class="image-hover-overlay-content text-center padding-2x">
                        <h2 class="text-thin">metaSTAAR</h2> 
                        <p>Meta-analysis of whole genome sequencing/whole exome sequencing (WGS/WES) studies provides an attractive solution to the problem of collecting large sample sizes for discovering rare variants associated with complex phenotypes. Existing rare variant meta-analysis approaches are not scalable to biobank-scale WGS data. Here we present MetaSTAAR, a powerful and resource-efficient rare variant meta-analysis framework for large-scale WGS/WES studies. MetaSTAAR accounts for relatedness and population structure, can analyze both quantitative and dichotomous traits and boosts the power of rare variant tests by incorporating multiple variant functional annotations. Through meta-analysis of four lipid traits in 30,138 ancestrally diverse samples from 14 studies of the Trans Omics for Precision Medicine (TOPMed) Program, we show that MetaSTAAR performs rare variant meta-analysis at scale and produces results comparable to using pooled data.</p> 
                      </div> 
                    </div> 
                    <img src="img/portfolio/metaSTAAR.png" alt="" title="metaSTAAR" />
                  </a>	
                </div>                
              </div>  
            </div>
            <div class="tab-item">
              <a class="tab-label">Epigenomics Research</a>
              <div class="tab-content lightbox-gallery">

                <div class="s-12 m-6 l-3">
                  <a class="image-with-hover-overlay image-hover-zoom" href="#">
                    <div class="image-hover-overlay background-primary"> 
                      <div class="image-hover-overlay-content text-center padding-2x">
                        <h2 class="text-thin">EBV Regulome</h2> 
                        <p>Epstein-Barr virus (EBV) transforms B cells to continuously proliferating lymphoblastoid cell lines (LCLs), which represent an experimental model for EBV-associated cancers. EBV nuclear antigens (EBNAs) and LMP1 are EBV transcriptional regulators that are essential for LCL establishment, proliferation, and survival. Starting with the 3D genome organization map of LCL, we constructed a comprehensive EBV regulome encompassing 1,992 viral/cellular genes and enhancers. Approximately 30% of genes essential for LCL growth were linked to EBV enhancers. Deleting EBNA2 sites significantly reduced their target gene expression. Additional EBV super-enhancer (ESE) targets included MCL1, IRF4, and EBF. MYC ESE looping to the transcriptional stat site of MYC was dependent on EBNAs. Deleting MYC ESEs greatly reduced MYC expression and LCL growth.</p> 
                      </div> 
                    </div> 
                    <img src="img/portfolio/EBVregulome.png" alt="" title="EBV regulome" />
                  </a>	
                </div>
                
                <div class="s-12 m-6 l-3">
                  <a class="image-with-hover-overlay image-hover-zoom" href="#">
                    <div class="image-hover-overlay background-primary"> 
                      <div class="image-hover-overlay-content text-center padding-2x">
                        <h2 class="text-thin">EBV Super Enhancers</h2> 
                        <p>Super-enhancers are clusters of gene-regulatory sites bound by multiple transcription factors that govern cell transcription, development, phenotype, and oncogenesis. By examining Epstein-Barr virus (EBV)-transformed lymphoblastoid cell lines (LCLs), we identified four EBV oncoproteins and five EBV-activated NF-κB subunits co-occupying ∼1,800 enhancer sites. Of these, 187 had markedly higher and broader histone H3K27ac signals, characteristic of super-enhancers, and were designated "EBV super-enhancers." EBV super-enhancer-associated genes included the MYC and BCL2 oncogenes, which enable LCL proliferation and survival. EBV super-enhancers were enriched for B cell transcription factor motifs and had high co-occupancy of STAT5 and NFAT transcription factors (TFs). EBV super-enhancer-associated genes were more highly expressed than other LCL genes.  </p> 
                      </div> 
                    </div> 
                    <img src="img/portfolio/EBV_Super_Enhancers.png" alt="" title="EBV Super Enhancers" />
                  </a>	
                </div>
                
                <div class="s-12 m-6 l-3">
                  <a class="image-with-hover-overlay image-hover-zoom" href="#">
                    <div class="image-hover-overlay background-primary"> 
                      <div class="image-hover-overlay-content text-center padding-2x">
                        <h2 class="text-thin">super-enhancer eRNAs</h2> 
                        <p>Epstein-Barr virus (EBV) super-enhancers (ESEs) are essential for lymphoblastoid cell (LCL) growth and survival. Reanalyses of LCL global run-on sequencing (Gro-seq) data found abundant enhancer RNAs (eRNAs) being transcribed at ESEs. Inactivation of ESE components, EBV nuclear antigen 2 (EBNA2) and bromodomain-containing protein 4 (BRD4), significantly decreased eRNAs at ESEs -428 and -525 kb upstream of the MYC oncogene transcription start site (TSS). shRNA knockdown of the MYC -428 and -525 ESE eRNA caused LCL growth arrest and reduced cell growth. Furthermore, MYC ESE eRNA knockdown also significantly reduced MYC expression, ESE H3K27ac signals, and MYC ESEs looping to MYC TSS. These data indicate that ESE eRNAs strongly affect cell gene expression and enable LCL growth.</p> 
                      </div> 
                    </div> 
                    <img src="img/portfolio/eRNA.png" alt="" title="super-enhancer eRNAs" />
                  </a>	
                </div>
                
                <div class="s-12 m-6 l-3">
                  <a class="image-with-hover-overlay image-hover-zoom" href="#">
                    <div class="image-hover-overlay background-primary"> 
                      <div class="image-hover-overlay-content text-center padding-2x">
                        <h2 class="text-thin">EBNALP</h2> 
                        <p>Epstein-Barr virus (EBV) nuclear antigens EBNALP (LP) and EBNA2 (E2) are coexpressed in EBV-infected B lymphocytes and are critical for lymphoblastoid cell line outgrowth. LP removes NCOR and RBPJ repressive complexes from promoters, enhancers, and matrix-associated deacetylase bodies, whereas E2 activates transcription from distal enhancers. LP ChIP-seq analyses identified 19,224 LP sites of which ~50% were ± 2 kb of a transcriptional start site. LP sites were enriched for B-cell transcription factors (TFs), YY1, SP1, PAX5, BATF, IRF4, ETS1, RAD21, PU.1, CTCF, RBPJ, ZNF143, SMC3, NFκB, TBLR, and EBF. E2 sites were also highly enriched for LP-associated cell TFs and were more highly occupied by RBPJ and EBF. LP sites were highly marked by H3K4me3, H3K27ac, H2Az, H3K9ac, RNAPII, and P300, indicative of activated transcription. </p> 
                      </div> 
                    </div> 
                    <img src="img/portfolio/EBNALP.png" alt="" title="Portfolio Image 4" />
                  </a>	
                </div>
                

              </div>  
            </div>
            <div class="tab-item">
              <a class="tab-label">Human-Microbe Interactions</a>
              <div class="tab-content lightbox-gallery">
                
                <div class="s-12 m-6 l-3">
                  <a class="image-with-hover-overlay image-hover-zoom" href="#">
                    <div class="image-hover-overlay background-primary"> 
                      <div class="image-hover-overlay-content text-center padding-2x">
                        <h2 class="text-thin">Comparative analysis of PPI</h2> 
                        <p>The Jaccard coefficient, precision and recall between H37Rv PPI datasets in STRING database predicted by different methods and the H37Rv B2H PPI dataset (benchmark).</p> 
                      </div> 
                    </div> 
                    <img src="img/portfolio/mtbppi1.png" alt="" title="Portfolio Image 9" />
                  </a>	
                </div>
                <div class="s-12 m-6 l-3">
                  <a class="image-with-hover-overlay image-hover-zoom" href="#">
                    <div class="image-hover-overlay background-primary"> 
                      <div class="image-hover-overlay-content text-center padding-2x">
                        <h2 class="text-thin">IntPath–integrated pathway database</h2> 
                        <p>We have overcome in IntPath the issues of compatibility, consistency, and comprehensiveness that often hamper effective use of pathway databases. We have included four organisms in the current release of IntPath. Our methodology and programs described in this work can be easily applied to other organisms.</p> 
                      </div> 
                    </div> 
                    <img src="img/portfolio/intpath1.png" alt="" title="Portfolio Image 10" />
                  </a>	
                </div>
                <div class="s-12 m-6 l-3">
                  <a class="image-with-hover-overlay image-hover-zoom" href="#">
                    <div class="image-hover-overlay background-primary"> 
                      <div class="image-hover-overlay-content text-center padding-2x">
                        <h2 class="text-thin">Homology-Based Prediction</h2>
               
                        <p>We develop a stringent homology-based prediction approach by taking into account (i) differences between eukaryotic and prokaryotic proteins and (ii) differences between inter-species and intra-species PPI interfaces. We compare our stringent homology-based approach to a conventional homology-based approach for predicting host-pathogen PPIs, based on cellular compartment distribution analysis, disease gene list enrichment analysis, pathway enrichment analysis and functional category enrichment analysis. These analyses support the validity of our prediction result, and clearly show that our approach has better performance in predicting H. sapiens-M. tuberculosis H37Rv PPIs. Using our stringent homology-based approach, we have predicted a set of highly plausible H. sapiens-M. tuberculosis H37Rv PPIs which might be useful for many of related studies. Based on our analysis of the H. sapiens-M. tuberculosis H37Rv PPI network predicted by our stringent homology-based approach, we have discovered several interesting properties which are reported here for the first time. We find that both host proteins and pathogen proteins involved in the host-pathogen PPIs tend to be hubs in their own intra-species PPI network. Also, both host and pathogen proteins involved in host-pathogen PPIs tend to have longer primary sequence, tend to have more domains, tend to be more hydrophilic, etc. And the protein domains from both host and pathogen proteins involved in host-pathogen PPIs tend to have lower charge, and tend to be more hydrophilic.</p>
                      </div> 
                    </div> 
                    <img src="img/portfolio/Homology.png" alt="" title="Homology-Based Prediction" />
                  </a>	
                </div>
                <div class="s-12 m-6 l-3">
                  <a class="image-with-hover-overlay image-hover-zoom" href="#">
                    <div class="image-hover-overlay background-primary"> 
                      <div class="image-hover-overlay-content text-center padding-2x">
                        <h2 class="text-thin">DDI-Based Prediction</h2> 
                        <p>We develop a stringent DDI-based prediction approach with emphasis on (i) differences between the specific domain sequences on annotated regions of proteins under the same domain ID and (ii) calculation of the interaction strength of predicted PPIs based on the interacting residues in their interaction interfaces. We compare our stringent DDI-based approach to a conventional DDI-based approach for predicting PPIs based on gold standard intra-species PPIs and coherent informative Gene Ontology terms assessment. The assessment results show that our stringent DDI-based approach achieves much better performance in predicting PPIs than the conventional approach. Using our stringent DDI-based approach, we have predicted a small set of reliable H. sapiens-M. tuberculosis H37Rv PPIs which could be very useful for a variety of related studies. </p>
                      </div> 
                    </div> 
                    <img src="img/portfolio/DDI.png" alt="" title="DDI-Based Prediction" />
                  </a>	
                </div>
              </div>  
            </div>
         </div>
      </section>
      
   

    </main>
    
 <!-- FOOTER -->
    <footer>
      <!-- Contact Us -->
      <div class="background-primary padding text-center">
        <p class="h1">+1-617-903-8599</p>                                                                        
      </div>
      
      <!-- Main Footer -->
      <section class="background-dark full-width">
        <!-- Map -->
        <div class="s-12 m-12 l-6 margin-m-bottom-2x">
          <div class="s-12 grayscale center">
            <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2947.336408464034!2d-71.10544858456852!3d42.337455079186126!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x89e37a3486a75e6f%3A0x92ff2e2c3145c5eb!2s677%20Huntington%20Ave%2C%20Boston%2C%20MA%2002115%2C%20USA!5e0!3m2!1sen!2sph!4v1651295501508!5m2!1sen!2sph" width="600" height="450" style="border:0;" allowfullscreen="" loading="lazy"></iframe>
          </div>
        </div>
        
        <!-- Collumn 2 -->
        <div class="s-12 m-12 l-6 margin-m-bottom-2x">
          <div class="padding-2x">
            <div class="line">              
              <div class="float-left">
                  <i class="icon-sli-location-pin text-primary icon3x"></i>
                </div>
                <div class="margin-left-70 margin-bottom-30">
                  <h3 class="margin-bottom-0">Office</h3>
                  <p>677 Huntington Ave<br>
                     Boston / MA / U.S.A /02115
                  </p>               
                </div>
                <div class="float-left">
                  <i class="icon-sli-envelope text-primary icon3x"></i>
                </div>
                <div class="margin-left-70 margin-bottom-30">
                  <h3 class="margin-bottom-0">E-mail</h3>
                  <p>hufengzhou@g.harvard.edu<br>
                  </p>              
                </div>
                <div class="float-left">
                  <i class="icon-sli-phone text-primary icon3x"></i>
                </div>
                <div class="margin-left-70">
                  <h3 class="margin-bottom-0">Phone Numbers</h3>
                  <p>+1-617-903-8599<br>

                  </p>             
                </div>
            </div>
          </div>
        </div>  
      </section>
      <hr class="break margin-top-bottom-0" style="border-color: rgba(0, 38, 51, 0.80);">
      
      <!-- Bottom Footer -->
      <section class="padding background-dark full-width">
        <div class="s-12 l-6">
          <p class="text-size-12">Copyright 2023, Hufeng Zhou</p>
          <p class="text-size-12"></p>
        </div>
        <div class="s-12 l-6">
          <a class="right text-size-12" href="http://hzhou.org" title="hzhou - lightweight responsive framework">Design and coding<br> by Hufeng Zhou</a>
        </div>
      </section>
    </footer>
    <script type="text/javascript" src="js/responsee.js"></script>
    <script type="text/javascript" src="owl-carousel/owl.carousel.js"></script>
    <script type="text/javascript" src="js/template-scripts.js"></script>
  </body>
</html>
