<!DOCTYPE html>
<html lang="en-US">

<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>Hufeng Zhou Official Website</title>
  <link rel="stylesheet" href="css/components.css">
  <link rel="stylesheet" href="css/icons.css">
  <link rel="stylesheet" href="css/responsee.css">
  <link rel="stylesheet" href="owl-carousel/owl.carousel.css">
  <link rel="stylesheet" href="owl-carousel/owl.theme.css">
  <link rel="stylesheet" href="css/template-style.css">
  <link href='https://fonts.googleapis.com/css?family=Playfair+Display&subset=latin,latin-ext' rel='stylesheet'
    type='text/css'>
  <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,700,800&subset=latin,latin-ext' rel='stylesheet'
    type='text/css'>
  <script type="text/javascript" src="js/jquery-1.8.3.min.js"></script>
  <script type="text/javascript" src="js/jquery-ui.min.js"></script>
</head>


<body class="size-1140">
  <!-- HEADER -->
  <header role="banner" class="position-absolute">
    <!-- Top Navigation -->
    <nav class="background-transparent background-transparent-hightlight full-width sticky">
      <div class="s-12 l-2">
        <a href="index.html" class="logo">
          <!-- Logo White Version -->
          <img class="logo-white" src="img/logo-dark.png" alt="">
          <!-- Logo Dark Version -->
          <img class="logo-dark" src="img/logo-dark.png" alt="">
        </a>
      </div>
      <div class="top-nav s-12 l-10">
        <p class="nav-text"></p>
        <ul class="right chevron">
          <li><a href="index.html">Home</a></li>
          <li><a href="about.html">About</a></li>
          <li><a href="publications.html">Publications</a></li>
          <li>
            <a>Projects</a>
            <ul>
              <li>
                <a href="Epigenomics.html">Epigenomics</a>
                <ul>
                  <li>
                    <a href="EBV_Oncoproteins.html">EBV oncoproteins</a>
                  </li>
                  <li>
                    <a href="EBV_Super_Enhancers.html">Super Enhancers</a>
                  </li>
                  <li>
                    <a href="EBV_Regulome.html">EBV Regulome</a>
                  </li>
                </ul>
              </li>

              <li>
                <a href="Population_Genetics.html">Population Genetics</a>
                <ul>
                  <li>
                    <a href="FAVOR.html">Functional Annotation Database</a>
                  </li>
                  <li>
                    <a href="STAAR.html">STAAR</a>
                  </li>
                </ul>
              </li>

              <li>
                <a href="PPIs.html">Protein-Protein Interactions</a>
                <ul>
                  <li>
                    <a href="IntraPPIs.html">Intra-Species PPIs</a>
                  </li>
                  <li>
                    <a href="Host-Pathogen-PPIs.html">Host-Pathogen PPIs</a>
                  </li>
                  <li>
                    <a href="Protein-functions.html">Protein Functions</a>
                  </li>
                </ul>
              </li>

            </ul>
          </li>


          <li><a href="Software.html">Software</a></li>
          <li><a href="contact.html">Contact</a></li>
        </ul>
      </div>
    </nav>
  </header>

  <!-- MAIN -->
  <main role="main">
    <!-- Content -->
    <article>
      <header class="section background-dark">
        <div class="line">
          <h1 class="text-white margin-top-bottom-40 text-size-60 text-line-height-1">Publications</h1>
          <p class="margin-bottom-0 text-size-16"> Dr. Zhou's diverse research interests enable him to tackle complex
            biological questions from different angles and gain a comprehensive understanding of the mechanisms
            underlying human diseases. His expertise in population genetics allows him to investigate the genetic basis
            of complex diseases in different populations, while his knowledge in epigenomics and proteomics allows him
            to examine the functional consequences of genetic variations at the molecular level. Additionally, his
            expertise in pathway analysis helps to reveal the interconnectedness of biological processes and the
            potential targets for drug development. </p>
        </div>
      </header>
      <section class="section background-white">
        <div class="line">
          <h2 class="text-size-40 margin-bottom-30"></h2>
          <hr class="break-small background-primary margin-bottom-30">
          <p class="margin-bottom-40">

          <div class="margin">
            <br> <strong>Selective Publications (* means equal contribution).</strong>
            <br>
            <br>1) Zhou H, Arapoglou T, Li X, Li Z, Zheng X, Moore JE, Asok A, Kumar S, Blue EE, Buyske S, Cox N,… Lin
            X. FAVOR: functional annotation of variants online resource and annotator for vari-ation across the human
            genome. Nucleic Acids Res. 2022 Nov 9; gkac966. doi: 10.1093/nar/gkac966.
            <br>
            <br>2) Wang C, Zhou H*, Xue Y, Liang J, Narita Y, Gerdt C, Zheng AY, Jiang R, Trudeau S, Peng CW, Gewurz BE.
            Epstein-Barr Virus Nuclear Antigen Leader Protein Coactivates EP300. Journal of virology. 2018 May
            1;92(9):e02155-17.
            <br>
            <br>3) Jiang S, Zhou H*, Liang J, Gerdt C, Wang C, Ke L, Schmidt SC, Narita Y, Ma Y, Wang S, Colson T. The
            Epstein-Barr virus regulome in lymphoblastoid cells. Cell host & microbe. 2017 Oct 11;22(4):561-73.
            <br>
            <br>4) Zhou H, Schmidt SC, Jiang S, Willox B, Bernhardt K, Liang J, Johannsen EC, Kharchenko P, Gewurz BE,
            Kieff E, Zhao B. Epstein-Barr virus oncoprotein super-enhancers control B cell growth. Cell host & microbe.
            2015 Feb 11;17(2):205-16.
            <br>
            <br>5) Portal D*, Zhou H*, Zhao B, Kharchenko PV, Lowry E, Wong L, Quackenbush J, Holloway D, Jiang S, Lu Y,
            Kieff E. Epstein-Barr virus nuclear antigen leader protein localizes to promoters and enhancers with cell
            transcription factors and EBNA2. Proceedings of the National Academy of Sciences. 2013 Nov
            12;110(46):18537-42. doi:10.1073/pnas.1317608110.
          </div>

          <div class="margin">          
          <br><strong>Other Publications (* means equal contribution)</strong>
          <br>
          <br>6) Granot-Hershkovitz E, Sun Q, Argos M, Zhou H, Lin X, Browning SR, Sofer T. AFA: Ancestry-specific
          allele frequency estimation in admixed populations: The Hispanic Community Health Study/Study of Latinos.
          Human Genetics and Genomics Advances. 2022 Apr 14;3(2):100096.
          <br>
          <br>7) Li X, Li Z, Zhou H, Gaynor SM, Liu Y, Chen H, Sun R, Dey R, Arnett DK, Aslibekyan S, Ballantyne CM.
          Dynamic incorporation of multiple in silico functional annotations empowers rare variant association analysis
          of large whole-genome sequencing studies at scale. Nature genetics. 2020 Sep;52(9):969-83.
          <br>
          <br>8) Li X, Yung G, Zhou H, Sun R, Li Z, Hou K, Zhang MJ, Liu Y, Arapoglou T, Wang C, Ionita-Laza I. A
          multi-dimensional integrative scoring framework for predicting functional variants in the human genome. The
          American Journal of Human Genetics. 2022 Mar 3;109(3):446-56.
          <br>
          <br>9) Sun R, Xu M, Li X, Gaynor S, Zhou H, Li Z, Bossé Y, Lam S, Tsao MS, Tardon A, Chen C. Integration of
          multiomic annotation data to prioritize and characterize inflammation and immune‐related risk variants in
          squamous cell lung cancer. Genetic epidemiology. 2021 Feb;45(1):99-114.
          <br>
          <br>10) Li Z, Li X, Zhou H, Gaynor SM, Selvaraj MS, Arapoglou T, Quick C, Liu Y, Chen H, Sun R, Dey R. A framework
          for detecting noncoding rare variant associations of large-scale whole-genome sequencing studies. Nature
          Methods. 2022 Oct 27. doi: 10.1038/s41592-022-01640-x.
          <br>
          <br>11) Byun J, Han Y, Li Y, Xia J, Long E, Choi J, Xiao X, Zhu M, … Zhou H, …, Bossé Y. Cross-ancestry
          genome-wide meta-analysis of 61,047 cases and 947,237 controls identifies new susceptibility loci contributing
          to lung cancer. Nature genetics. 2022 Aug;54(8):1167-77.
          <br>
          <br>12) Wainschtein P, Jain D, Zheng Z, Cupples LA, Shadyab AH, McKnight B, Shoemaker BM, Mitchell BD, Psaty BM,
          Kooperberg C, Liu CT. Assessing the contribution of rare variants to complex trait heritability from
          whole-genome sequence data. Nature Genetics. 2022 Mar;54(3):263-73.
          <br>
          <br>13) Cade BE, Lee J, Sofer T, Wang H, Zhang M, Chen H, Gharib SA, Gottlieb DJ, Guo X, Lane JM, Liang J.
          Whole-genome association analyses of sleep-disordered breathing phenotypes in the NHLBI TOPMed program. Genome
          medicine. 2021 Dec;13(1):1-7.
          <br>
          <br>14) Wang H, Kurniansyah N, Cade BE, Goodman MO, Chen H, Gottlieb DJ, Gharib SA, Purcell SM, Lin X, Saxena R,
          Zhu X. Upregulated heme biosynthesis increases obstructive sleep apnea severity: a pathway-based Mendelian
          randomization study. Scientific reports. 2022 Jan 27;12(1):1-1.
          <br>
          <br>15) Wang L, Laing J, Yan B, Zhou H, Ke L, Wang C, Narita Y, Zhang Z, Olson MR, Afzali B, Zhao B. Epstein-Barr
          virus episome physically interacts with active regions of the host genome in lymphoblastoid cells. Journal of
          Virology. 2020 Nov 23;94(24):e01390-20.
          <br>
          <br>16) Kessler MD, Loesch DP, Perry JA, Heard-Costa NL, Taliun D, Cade BE, Wang H, Daya M, Ziniti J, Datta S,
          Celedón JC. De novo mutations across 1,465 diverse genomes reveal mutational insights and reductions in the
          Amish founder population. Proceedings of the National Academy of Sciences. 2020 Feb 4;117(5):2560-9.
          <br>
          <br>17) Li Z, Li X, Liu Y, Shen J, Chen H, Zhou H, Morrison AC, Boerwinkle E, Lin X. Dynamic Scan Procedure for
          Detecting Rare-Variant Association Regions in Whole-Genome Sequencing Studies. The American Journal of Human
          Genetics. 2019 May 2;104(5):802-14.
          <br>
          <br>18) Hua K, Li Y, Zhou H, Hu X, Chen Y, He R, Luo R, Zhou R, Bi D, Jin H. Haemophilus parasuis in-fection
          disrupts adherens junctions and initializes EMT dependent on canonical Wnt/β-catenin signaling pathway.
          Frontiers in cellular and infection microbiology. 2018 Sep 12;8:324.
          <br>
          <br>19) Ke L, Zhou H, Wang C, Xiong G, Xiang Y, Ling Y, Khabir A, Tsao GS, Zeng Y, Zeng M, Busson P.
          Nasopharyngeal carcinoma super-enhancer–driven ETV6 correlates with prognosis. Proceedings of the National
          Academy of Sciences. 2017 Aug 22:201705236.
          <br>
          <br>20) Ma B, Hua K, Zhou S, Zhou H, Chen Y, Luo R, Bi D, Zhou R, He Q, Jin H. Haemophilus parasuis infection
          activates NOD1/2-RIP2 signaling pathway in PK-15 cells. Developmental & Comparative Immunology. 2018 Feb
          1;79:158-65.
          <br>
          <br>21) Nayar U, Sadek J, Reichel J, Hernandez-Hopkins D, Akar G, Barelli PJ, Sahai MA, Zhou H, Totonchy J,
          Jayabalan D, Niesvizky R. Identification of a nucleoside analog active against adenosine kinase–expressing
          plasma cell malignancies. The Journal of clinical investigation. 2017 Jun 1;127(6):2066-80.
          <br>
          <br>22) Minamitani T, Ma Y, Zhou H, Kida H, Tsai CY, Obana M, Okuzaki D, Fujio Y, Kumanogoh A, Zhao B, Kikutani H.
          Mouse model of Epstein-Barr virus LMP1-and LMP2A-driven germinal center B-cell lymphoproliferative disease.
          Proceedings of the National Academy of Sciences. 2017 Mar 28:201701836.
          <br>
          <br>23) Liang J, Zhou H, Gerdt C, Tan M, Colson T, Kaye KM, Kieff E, Zhao B. Epstein-Barr virus super-enhancer
          eRNAs are essential for MYC oncogene expression and lymphoblast proliferation. Proceedings of the National
          Academy of Sciences. 2016 Dec 6;113(49):14121-14126.
          <br>
          <br>24) Minamitani T, Yasui T, Ma Y, Zhou H, Okuzaki D, Tsai CY, Sakakibara S, Gewurz BE, Kieff E, Kikutani H.
          Evasion of affinity-based selection in germinal centers by Epstein-Barr virus LMP2A. Proceedings of the
          National Academy of Sciences. 2015 Sep 15;112(37):11612-7. doi: 10.1073/pnas.1514484112.
          <br>
          <br>25) Hsu DJ, Feng M, Kothari R, Zhou H, Chen KP, Celi LA. The Association Between Indwelling Arterial Catheters
          and Mortality in Hemodynamically Stable Patients With Respiratory Failure: A Propensity Score Analysis. Chest.
          2015 Dec;148(6):1470-6. doi: 10.1378/chest.15-0516.
          <br>
          <br>26) Greenfeld H, Takasaki K, Walsh MJ, Ersing I, Bernhardt K, Ma Y, Fu B, Ashbaugh CW, Cabo J, Mollo SB, Zhou
          H, Li S, Gewurz BE. TRAF1 Coordinates Polyubiquitin Signaling to Enhance Epstein-Barr Virus LMP1-Mediated
          Growth and Survival Pathway Activation. PLoS Pathog. 2015 May 21;11(5):e1004890.
          doi:10.1371/journal.ppat.1004890.
          <br>
          <br>27) Schmidt SC, Jiang S, Zhou H, Willox B, Holthaus AM, Kharchenko PV, Johannsen EC, Kieff E, Zhao B.
          Epstein-Barr virus nuclear antigen 3A partially coincides with EBNA3C genome-wide and is tethered to DNA
          through BATF complexes. Proceedings of the National Academy of Sciences. 2015 Jan 13;112(2):554-9. doi:
          10.1073/pnas.1422580112.
          <br>
          <br> 28) Zhao B, Barrera LA, Ersing I, Willox B, Schmidt SC, Greenfeld H, Zhou H, Mollo SB, Shi TT, Takasaki K,
          Jiang S, Cahir-McFarland E, Kellis M, Bulyk ML, Kieff E, Gewurz BE. The NF-κB genomic landscape in
          lymphoblastoid B cells. Cell Reports. 2014 Sep 11;8(5):1595-606. doi: 10.1016/j.celrep.2014.07.037.
          <br>
          <br>29) Xiong G, Zhang B, Huang MY, Zhou H, Chen LZ, Feng QS, Luo X, Lin HJ, Zeng YX. Epstein-Barr virus (EBV)
          infection in Chinese children: a retrospective study of age-specific prevalence. PLoS One. 2014 Jun
          10;9(6):e99857. doi:10.1371/journal.pone.0099857.
          <br>
          <br>30) Zhou H, Gao S, Nguyen NN, Fan M, Jin J, Liu B, Zhao L, Xiong G, Tan M, Li S, Wong L. Stringent
          homology-based prediction of H. sapiens-M. tuberculosis H37Rv protein-protein interactions. Biology Direct.
          2014 Apr 8;9:5. doi:10.1186/1745-6150-9-5.
          <br>
          <br>31) Jiang S, Willox B, Zhou H, Holthaus AM, Wang A, Shi TT, Maruo S, Kharchenko PV, Johannsen EC, Kieff E,
          Zhao B. Epstein-Barr virus nuclear antigen 3C binds to BATF/IRF4 or SPI1/IRF4 composite sites and recruits
          Sin3A to repress CDKN2A. Proceedings of the National Academy of Sciences. 2014 Jan 7;111(1):421-6. doi:
          10.1073/pnas.1321704111.
          <br>
          <br>32) Fan M, Low HS, Zhou H, Wenk MR, Wong L. LipidGO: database for lipid-related GO terms and applications.
          Bioinformatics. 2014 Apr 1;30(7):1043-4. doi: 10.1093/bioinformatics/btt689.
          <br>
          <br>33) Zhou H, Jin J, Wong L. Progress in computational studies of host-pathogen interactions. Journal of
          Bioinformatics and Computational Biology. 2013 Apr;11(2):1230001. doi: 10.1142/S0219720012300018. Review.
          <br>
          <br>34) Zhou H, Jin J, Zhang H, Yi B, Wozniak M, Wong L. IntPath--an integrated pathway gene relationship database
          for model organisms and important pathogens. BMC System Biology. 2012;6 Suppl 2:S2. doi:
          10.1186/1752-0509-6-S2-S2.
          <br>
          <br>35) Zhou H, Wong L. Comparative analysis and assessment of M. tuberculosis H37Rv protein-protein interaction
          datasets. BMC Genomics. 2011 Nov 30;12 Suppl 3:S20. doi: 10.1186/1471-2164-12-S3-S20.
          <br>
          <br>36) Xie Q, Jin H, Luo R, Wan Y, Chu J, Zhou H, Shi B, Chen H, Zhou R. Transcriptional responses of Haemophilus
          parasuis to iron-restriction stress in vitro. Biometals. 2009 Dec;22(6):907-16. doi:
          10.1007/s10534-009-9243-2.<br>
        </div>
        <div class="margin">
          <br><strong>Manuscripts in Preparation (* means equal contribution)</strong>
          <br>
          <br>1) Zhou H*, Li X, Li Z, Lin X. Functional Landscape of variants from large-scale whole genome sequencing
          studies. manuscript in preparation, intended to submit to Nature Communications.
          <br>
          <br>2) Zhou H*, Li X, Li Z, Lin X. Genotype consistency of large-scale whole-genome sequencing studies among
          different genotype callers. manuscript in preparation, intended to submit to AJHG.
          
        </p>
        </div>
        </div>


      </section>
    </article>
  </main>

  <!-- FOOTER -->
  <footer>
    <!-- Contact Us -->
    <div class="background-primary padding text-center">
      <p class="h1">+1-617-903-8599</p>
    </div>

    <!-- Main Footer -->
    <section class="background-dark full-width">
      <!-- Map -->
      <div class="s-12 m-12 l-6 margin-m-bottom-2x">
        <div class="s-12 grayscale center">
          <iframe
            src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2947.336408464034!2d-71.10544858456852!3d42.337455079186126!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x89e37a3486a75e6f%3A0x92ff2e2c3145c5eb!2s677%20Huntington%20Ave%2C%20Boston%2C%20MA%2002115%2C%20USA!5e0!3m2!1sen!2sph!4v1651295501508!5m2!1sen!2sph"
            width="1200" height="400" style="border:0;" allowfullscreen="" loading="lazy"></iframe>
        </div>
      </div>

      <!-- Collumn 2 -->
      <div class="s-12 m-12 l-6 margin-m-bottom-2x">
        <div class="padding-2x">
          <div class="line">
            <div class="float-left">
              <i class="icon-sli-location-pin text-primary icon3x"></i>
            </div>
            <div class="margin-left-70 margin-bottom-30">
              <h3 class="margin-bottom-0">Office</h3>
              <p>677 Huntington Ave<br>
                Boston / MA / U.S.A /02115
              </p>
            </div>
            <div class="float-left">
              <i class="icon-sli-envelope text-primary icon3x"></i>
            </div>
            <div class="margin-left-70 margin-bottom-30">
              <h3 class="margin-bottom-0">E-mail</h3>
              <p>hufengzhou@g.harvard.edu<br>
              </p>
            </div>
            <div class="float-left">
              <i class="icon-sli-phone text-primary icon3x"></i>
            </div>
            <div class="margin-left-70">
              <h3 class="margin-bottom-0">Phone Numbers</h3>
              <p>+1-617-903-8599<br>

              </p>
            </div>
          </div>
        </div>
      </div>
    </section>
    <hr class="break margin-top-bottom-0" style="border-color: rgba(0, 38, 51, 0.80);">

    <!-- Bottom Footer -->
    <section class="padding background-dark full-width">
      <div class="s-12 l-6">
        <p class="text-size-12">Copyright 2023, Hufeng Zhou</p>
        <p class="text-size-12"></p>
      </div>
      <div class="s-12 l-6">
        <a class="right text-size-12" href="http://hzhou.org" title="hzhou - lightweight responsive framework">Design
          and coding<br> by Hufeng Zhou</a>
      </div>
    </section>
  </footer>
  <script type="text/javascript" src="js/responsee.js"></script>
  <script type="text/javascript" src="owl-carousel/owl.carousel.js"></script>
  <script type="text/javascript" src="js/template-scripts.js"></script>
</body>

</html>
